src/PKSim.R/Services/IndividualFactory.cs

```diff
@@ -113,13 +113,7 @@ public DistributedParameterValueWithUnit[] DistributionsFor(IndividualCharacteri
 
       private OriginData originDataFrom(Core.Snapshots.OriginData originData)
       {
-         var coreOriginData = _originDataMapper.MapToModel(originData).Result;
```

Comment:
This was a bug here that needed to be fixed for R. Why do we required age but not height etc.. not consistent.